twilio/http/http_client.py

```diff
@@ -1,20 +1,30 @@
-from requests import Request, Session
+from requests import Request, Session, hooks
 
 from twilio.http import HttpClient, get_cert_file
 from twilio.http.response import Response
+import logging
+import json
+try:
+    from urllib.parse import urlencode
+except ImportError:
+    from urllib import urlencode
```

Comment:
you should import things via the `compat.py` module which handles version specific imports like this for you.